src/lifecycle/props-init.js

```diff
@@ -52,21 +54,23 @@ function createNativePropertyDefinition (name, opts) {
     let { oldValue } = propData;
 
     if (empty(oldValue)) {
+      // todo: the doc is incorrect:  When the property is initialised, oldValue will always be undefined
+      // we probabbly need to update the doc
```

Comment:
(thumbsup) I don't remember why we do null here instead of `undefined`, but maybe nulling everything is better since you can redefine `undefined`.